tests/std/tests/P0355R7_calendars_and_time_zones_formatting/test.cpp

```diff
@@ -0,0 +1,211 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <assert.h>
+#include <chrono>
+#include <concepts>
+#include <format>
+#include <iostream>
+#include <stdio.h>
+#include <string_view>
+#include <type_traits>
+#include <utility>
+
+using namespace std;
+using namespace chrono;
+
+#if defined(__cpp_lib_concepts) && !defined(__clang__) // TRANSITION, LLVM-48606

```

Comment:
As the entire test is guarded for concepts, perhaps this should simply use the `concepts_matrix` making the concepts guard unnecessary.